### PR TITLE
Replace this repo with a pointer to docs.github.com

### PR DESCRIPTION
**Pull request by hsenag**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,48 +2,7 @@
```
Comment (bmuskalla) [inline README.md:2]: ^^ this should not say tech preview anymore

File: README.md [hunk 2]
```diff
@@ -2,48 +2,7 @@
 
 <img width="128" alt="GitHub Copilot Logo" src="https://user-images.githubusercontent.com/28068/123712981-02676c80-d839-11eb-919a-96ee0c895e15.png">
```
Comment (bmuskalla) [inline README.md:3]: still the old logo

File: README.md [hunk 3]
```diff
@@ -2,48 +2,7 @@
...
 
-Welcome to the GitHub Copilot user community!
-In this repository, you can find documentation, walkthroughs, examples, and the latest resources you need to use GitHub Copilot.
+This repository was used for GitHub Copilot documentation during the technical preview, which has
```
Comment (bmuskalla) [inline README.md:5]: Should we try to be more positive and forward looking (drop `ended` and use something like the blog post itself `GitHub Copilot is generally available to all developers`)

**Review by bmuskalla — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (hsenag) [reply README.md:3]: Again thought it simpler jsut to reflect that this is the old world.

**Review by bmuskalla — APPROVED** [review]
I'm generally ok with this. I still think we should use the newer logo for all public-facing entities.

File: README.md (hunk 2 above)
Comment (hsenag) [reply README.md]: Dropped the logo too

**Review by jules-p — APPROVED** [review]
Looks good to me! 🚀

**Review by Hayderhammami — COMMENTED** [review]
Good

**Comment by Someshsw1109** [comment]
That's awesome

**Comment by allan1366** [comment]
来信收到，谢谢！张韶华

**Review by Hosamph — APPROVED** [review]

**Comment by allan1366** [comment]
来信收到，谢谢！张韶华

**Pull request merged**